github-profiles/kenichi-7356/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <title>Perfil profesional</title>
+</head>
+<body>
+    <div class="header">
+    <h1><strong>Hi There</strong></h1>
+    </div>
+    <hr>
+   <!-- <dl>
+        <a href="#about"><h3><dt>About me</dt></h3></a>
+        <a href="#week1"><h3><dt>Bootcamp first week</dt></h3></a>
+        <h3><dd>- Creating alias for use in terminal</dd></h3>
+        <h3><dd>- Most used command in terminal</dd></h3>
+        <h3><dd>- HTML</dd></h3>
+    </dl>
+    <hr>
+-->
+    <h3 id="about">About me:</h3>
+    <p>
+    <strong>My name is Alfonso Tovar</strong>, 🇨🇴 Colombian, in my career as a in <strong>Analysis and development of information systems
+    </strong>, I focus on<strong>specialized support</strong> to users in the use of ICT, but I like to learn topics about}
+    <b>programming language</b>, and 🧰<b>for development</b>
+    <i><dt>- 👀 Looking to work in development companies.</dt> </i>
+    <i><dt>- 💻 I'm currently learning about being a at<strong>bootcamp 2023 on the Twitch channel</strong></dt> </i>
+    <strong>//www.twitch.tv/guillermorodas</strong>.
+    <i><dt>- ✋I participated in the <strong>2022 mission</strong> program of the <strong>Ministerio de las TIC, Government of Colombia</strong></dt></i>
+    </p>
+<h3 id="week1" >📆 BootCamp week 1</h3>
```

Comment:
You have 2 `h3` tags on a row, and not `h2`. I mean: `h1`-`h3`-`h3`. It's recommendable follow some order with headings, but it's up to you according what you want communicate, the page's design....